actionpack/lib/action_controller/metal/live.rb

```diff
@@ -295,7 +297,7 @@ def log_error(exception)
       return unless logger
 
       logger.fatal do
-        message = "\n#{exception.class} (#{exception.message}):\n"
+        message = "\n#{exception.class} (#{exception.message}):\n".dup
```

Comment:
According to https://github.com/bbatsov/rubocop/pull/4586 dup is not the fastest way.